packages/psss-server/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@tupaia/psss-server",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Backend server for PSSS application",
+  "homepage": "https://github.com/beyondessential/tupaia",
+  "bugs": {
+    "url": "https://github.com/beyondessential/tupaia/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/beyondessential/tupaia"
+  },
+  "author": "Beyond Essential Systems <admin@tupaia.org> (https://beyondessential.com.au)",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "babel src --out-dir dist --source-maps --ignore 'src/__tests__/**' --extensions '.ts' --config-file '../../.babelrc-ts.js'",
+    "lint": "tsc; eslint . --ext .ts",
+    "lint:all": "yarn run lint \"src/**/*.{ts,jsx}\"",
+    "lint:fix": "yarn lint --fix",
+    "prestart": "npm run -s build",
+    "start": "node dist",
+    "start-dev": "../../scripts/bash/backendStartDev.sh 9998 -ts",
+    "start-verbose": "LOG_LEVEL=debug npm run start-dev",
+    "test": "jest",
+    "test:coverage": "jest --coverage",
+    "test:watch": "jest --watch"
+  },
+  "dependencies": {
+    "@babel/polyfill": "^7.0.0",
+    "@tupaia/access-policy": "3.0.0",
+    "@tupaia/database": "1.0.0",
+    "@tupaia/utils": "1.0.0",
+    "@types/client-sessions": "^0.8.0",
```

Comment:
I think `@types` dependencies are just required as `devDependencies`. There's also an argument for adding them to the `root` devDependencies, but I'm not too fussed 😄

Reply:
Oh right cool! Will update in the PR off this one, because I made the same mistake there